hooks/tk-houdini_actions.py

```diff
@@ -116,4 +137,46 @@ def _merge(self, path, sg_publish_data):
         hou.hipFile.merge(path,
                           node_pattern="*",
                           overwrite_on_conflict=False,
-                          ignore_load_warnings=False)       
+                          ignore_load_warnings=False) 
+
+
+    ##############################################################################################################
+    def _import(self, path, sg_publish_data):
+
+        import hou
+        app = self.parent
+
+        name = sg_publish_data.get("name", "alembic")
+        path = self.get_publish_path(sg_publish_data)
+
+        obj_context = hou.node("/obj")
+        app.log_debug("Creating geo: %s" % (name,))
+        geo_node = obj_context.createNode("geo", name)
+
+        # delete the default nodes created in the geo
+        for child in geo_node.children():
+            child.destroy()
+
+        app.log_debug("Creating alembic sop: %s" % (name,))
+        alembic_sop = geo_node.createNode("alembic", name)
+        app.log_debug("Setting alembic sop path: %s" % (path,))
+        alembic_sop.parm("fileName").set(path)
+        alembic_sop.parm("reload").pressButton()
+
+
+    ##############################################################################################################
+    def _import_archive(self, path, sg_publish_data):
+
+        import hou
+        app = self.parent
+
+        name = sg_publish_data.get("name", "alembicarchive")
+        path = self.get_publish_path(sg_publish_data)
+
+        obj_context = hou.node("/obj")
```

Comment:
I modified the code to first attempt to retrieve the current obj context and create the node there. If it can't create it there it falls back to top-level "/obj". Will resubmit for CR so you can see if what I did makes sense.